README.md

```diff
@@ -1,57 +1,69 @@
 # science-tots
 
-BEFORE YOU WRITE ANY OTHER CODE FOLLOW THESE DIRECTIONS
-These are the step by step directions with *everything* spelled out so we are all on the same page. 
-This is also the way we would like you to do all of your pull requests starting at number 5.
+**BEFORE YOU WRITE ANY OTHER CODE FOLLOW THESE DIRECTIONS**
+
+These are the step by step directions with *everything* spelled out so we are all on the same page. This is also the way we would like you to do all of your pull requests starting at number 5.
 
 1. Start by clicking on the 'Clone or download' button. Click the clipboard button to copy the link.
 
 2. Open VS Code and open the terminal in VS Code (it's at the very top).
 
 3. In the terminal you should hit enter after all of the following instructions:
-in windows command prompt: dir
-in windows power shell and mac:
-    > ls (to see which directory you want to put this project in)
-    > cd the_name_of_the_directory_you_want_to_open (for myself I keep everything code related in a projects folder)
-in mac:
-    > git clone COMMAND + V (to paste)
-in windows:
-    > git clone CTRL + V (to paste)
-both:    
-    > git cd science-tots
-    >git branch your_first_name
-    >git checkout your_first_name
-    >open README.md
+   - in windows command prompt: 
```

Comment:
I think you have your indentations backwards (see screenshot). Should be something like this I think:
- in windows command prompt:
    - dir

- in windows power shell and mac:
    -ls...

I use 4 spaces to create an indent in markdown

<img width="814" alt="Screen Shot 2019-06-07 at 3 17 29 PM" src="https://user-images.githubusercontent.com/49502059/59128325-bedef480-8937-11e9-836e-d8ef507c251b.png">

Reply:
I was trying to have all the things you actually type into the terminal be black dots, so that's why I formatted that way.